dbt/adapters/athena/impl.py

```diff
@@ -722,3 +722,24 @@ def get_columns_in_relation(self, relation: AthenaRelation) -> List[AthenaColumn
         return [
             AthenaColumn(column=c["Name"], dtype=c["Type"], table_type=table_type) for c in columns + partition_keys
         ]
+
+    @available
+    def delete_from_glue_catalog(self, relation: AthenaRelation):
+        schema_name = relation.schema
+        table_name = relation.table
+
+        conn = self.connections.get_thread_connection()
+        client = conn.handle
+
+        with boto3_client_lock:
+            glue_client = client.session.client("glue", region_name=client.region_name, config=get_boto3_config())
+
+        try:
+            glue_client.delete_table(DatabaseName=schema_name, Name=table_name)
+            logger.debug(f"Deleted table from glue catalog: {schema_name}.{table_name}")
+        except ClientError as e:
+            if e.response["Error"]["Code"] == "EntityNotFoundException":
+                logger.debug(f"Table '{schema_name}.{table_name}' does not exist and will not be deleted, ignoring")
```

Comment:
If we have the relation, we can use `relation.render()` instead of schema_name.table_name

Reply:
Thanks! This is now changed:

https://github.com/dbt-athena/dbt-athena/blob/d4135bee7e02b89f74cf103f914296ba3dab40d0/dbt/adapters/athena/impl.py#L739-L742